review/fileToReview.js

```diff
@@ -11,36 +11,23 @@ var domain = "bank.local.fr"
  * @param {Object} previousTransactions Previous page of transactions (To ckeck for dupes)
  * @return {Object} All transactions available on the page
  */
-async function fetchTransactions(fromDate, authorization, jws = null, id, page, previousTransactions) {
```

Comment:
Default parameters should be last

Reply:
ça a effectivement du sens : on pourrait donc appeler la fonction sans l'argument jws